app/controllers/users_controller.rb

```diff
@@ -843,15 +843,17 @@ def search_users
     topic_id = topic_id.to_i if topic_id
     topic_allowed_users = params[:topic_allowed_users] || false
 
-    if params[:group].present?
-      @group = Group.find_by(name: params[:group])
+    group_names = params[:groups] || []
```

Comment:
I think we need to update the request specs to reflect the change here.

Reply:
Done 👍